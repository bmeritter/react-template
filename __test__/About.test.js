import React from 'react';
import renderer from 'react-test-renderer';
import About from "../src/components/about/About";

describe('About component', () => {
	it('snapshot', () => {
		const comp = renderer.create( <About />)
		expect(comp).toMatchSnapshot();
	});
})
