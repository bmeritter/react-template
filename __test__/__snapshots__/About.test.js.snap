// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component snapshot 1`] = `
<div>
  about
</div>
`;
